toolkit/trace/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/apache/skywalking-go/toolkit/trace
```

Comment:
I think we only need to let the `github.com/apache/skywalking-go/toolkit` as a new module, `trace` is only a part of the `toolkit`. After this, you can add the `metrics` or `logs` if you have interests.